src/sentry/lang/javascript/processor.py

```diff
@@ -657,6 +657,11 @@ def process_frame(self, frame, stacktrace_info, idx):
         if not frame.get('abs_path'):
             return
 
+        is_react_native = False
+        if frame['abs_path'].startswith('react-native:'):
```

Comment:
where does react-native: come from here. Is this something we emit manually in the client?